packages/amplication-server/src/core/deployment/deployment.service.ts

```diff
@@ -98,9 +101,49 @@ export class DeploymentService {
     private readonly prisma: PrismaService,
     private readonly deployerService: DeployerService,
     private readonly actionService: ActionService,
+    private readonly environmentService: EnvironmentService,
+
     @Inject(WINSTON_MODULE_PROVIDER) private readonly logger: winston.Logger
   ) {}
 
+  async autoDeployToSandbox(build: Build): Promise<Deployment> {
+    const sandboxEnvironment = await this.environmentService.getDefaultEnvironment(
+      build.appId
+    );
+
+    const deployment = (await this.prisma.deployment.create({
```

Comment:
Use `this.create` method instead

Reply:
The two methods Create, and AutoDeply uses a different set of parameters and each method work differenly.
In Create we are also creating a new "Action" and in "AutoDeploy" we are connecting to an existing "Action"
Also, in Create we are adding all the initial steps like "add to queue" and deploy info, in AutoDeploy we only add a step to the action